README.md

```diff
@@ -385,8 +385,8 @@ By default, when a script fails (meaning it returned with a non-zero exit code),
 all scripts that are already running are allowed to finish, but new scripts are
```

Comment:
Clarifying, `default` and `continue` won't break a `watch` mode script, while `kill` will stop the `watch` on failure?

Reply:
A script failure will never cause a watch command to completely exit. The next time you change a file, it will try again. So `WIREIT_FAILURES` affects how quickly one iteration ends, which unblocks the next iteration from starting if a file changed.

Reply:
Gotcha!